package edu.towson.cis.cosc442.project4.coffeemaker;

import org.junit.*;
import static org.junit.Assert.*;

/**
 * The class <code>MainTest</code> contains tests for the class <code>{@link Main}</code>.
 *
 * @generatedBy CodePro at 3/22/18 1:54 PM
 * @author anthonyvisalli
 * @version $Revision: 1.0 $
 */
public class MainTest {
	/**
	 * Run the void addInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddInventory_1()
		throws Exception {

		Main.addInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addInventory(Main.java:206)
	}

	/**
	 * Run the void addInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddInventory_2()
		throws Exception {

		Main.addInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addInventory(Main.java:206)
	}

	/**
	 * Run the void addInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddInventory_3()
		throws Exception {

		Main.addInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addInventory(Main.java:206)
	}

	/**
	 * Run the void addInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddInventory_4()
		throws Exception {

		Main.addInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addInventory(Main.java:206)
	}

	/**
	 * Run the void addInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddInventory_5()
		throws Exception {

		Main.addInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addInventory(Main.java:206)
	}

	/**
	 * Run the void addInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddInventory_6()
		throws Exception {

		Main.addInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addInventory(Main.java:206)
	}

	/**
	 * Run the void addInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddInventory_7()
		throws Exception {

		Main.addInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addInventory(Main.java:206)
	}

	/**
	 * Run the void addInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddInventory_8()
		throws Exception {

		Main.addInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addInventory(Main.java:206)
	}

	/**
	 * Run the void addInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddInventory_9()
		throws Exception {

		Main.addInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addInventory(Main.java:206)
	}

	/**
	 * Run the void addInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddInventory_10()
		throws Exception {

		Main.addInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addInventory(Main.java:206)
	}

	/**
	 * Run the void addInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddInventory_11()
		throws Exception {

		Main.addInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addInventory(Main.java:206)
	}

	/**
	 * Run the void addInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddInventory_12()
		throws Exception {

		Main.addInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addInventory(Main.java:206)
	}

	/**
	 * Run the void addInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddInventory_13()
		throws Exception {

		Main.addInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addInventory(Main.java:206)
	}

	/**
	 * Run the void addInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddInventory_14()
		throws Exception {

		Main.addInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addInventory(Main.java:206)
	}

	/**
	 * Run the void addInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddInventory_15()
		throws Exception {

		Main.addInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addInventory(Main.java:206)
	}

	/**
	 * Run the void addInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddInventory_16()
		throws Exception {

		Main.addInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addInventory(Main.java:206)
	}

	/**
	 * Run the void addRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddRecipe_1()
		throws Exception {

		Main.addRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addRecipe(Main.java:83)
	}

	/**
	 * Run the void addRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddRecipe_2()
		throws Exception {

		Main.addRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addRecipe(Main.java:83)
	}

	/**
	 * Run the void addRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddRecipe_3()
		throws Exception {

		Main.addRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addRecipe(Main.java:83)
	}

	/**
	 * Run the void addRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddRecipe_4()
		throws Exception {

		Main.addRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addRecipe(Main.java:83)
	}

	/**
	 * Run the void addRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddRecipe_5()
		throws Exception {

		Main.addRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addRecipe(Main.java:83)
	}

	/**
	 * Run the void addRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddRecipe_6()
		throws Exception {

		Main.addRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addRecipe(Main.java:83)
	}

	/**
	 * Run the void addRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddRecipe_7()
		throws Exception {

		Main.addRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addRecipe(Main.java:83)
	}

	/**
	 * Run the void addRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddRecipe_8()
		throws Exception {

		Main.addRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addRecipe(Main.java:83)
	}

	/**
	 * Run the void addRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddRecipe_9()
		throws Exception {

		Main.addRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addRecipe(Main.java:83)
	}

	/**
	 * Run the void addRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddRecipe_10()
		throws Exception {

		Main.addRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addRecipe(Main.java:83)
	}

	/**
	 * Run the void addRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddRecipe_11()
		throws Exception {

		Main.addRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addRecipe(Main.java:83)
	}

	/**
	 * Run the void addRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddRecipe_12()
		throws Exception {

		Main.addRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addRecipe(Main.java:83)
	}

	/**
	 * Run the void addRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddRecipe_13()
		throws Exception {

		Main.addRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addRecipe(Main.java:83)
	}

	/**
	 * Run the void addRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddRecipe_14()
		throws Exception {

		Main.addRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addRecipe(Main.java:83)
	}

	/**
	 * Run the void addRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddRecipe_15()
		throws Exception {

		Main.addRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addRecipe(Main.java:83)
	}

	/**
	 * Run the void addRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testAddRecipe_16()
		throws Exception {

		Main.addRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.addRecipe(Main.java:83)
	}

	/**
	 * Run the void checkInventory() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testCheckInventory_1()
		throws Exception {

		Main.checkInventory();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.checkInventory(Main.java:211)
	}

	/**
	 * Run the void deleteRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testDeleteRecipe_1()
		throws Exception {

		Main.deleteRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.deleteRecipe(Main.java:92)
	}

	/**
	 * Run the void deleteRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testDeleteRecipe_2()
		throws Exception {

		Main.deleteRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.deleteRecipe(Main.java:92)
	}

	/**
	 * Run the void deleteRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testDeleteRecipe_3()
		throws Exception {

		Main.deleteRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.deleteRecipe(Main.java:92)
	}

	/**
	 * Run the void deleteRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testDeleteRecipe_4()
		throws Exception {

		Main.deleteRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.deleteRecipe(Main.java:92)
	}

	/**
	 * Run the void deleteRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testDeleteRecipe_5()
		throws Exception {

		Main.deleteRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.deleteRecipe(Main.java:92)
	}

	/**
	 * Run the void deleteRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testDeleteRecipe_6()
		throws Exception {

		Main.deleteRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.deleteRecipe(Main.java:92)
	}

	/**
	 * Run the void deleteRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testDeleteRecipe_7()
		throws Exception {

		Main.deleteRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.deleteRecipe(Main.java:92)
	}

	/**
	 * Run the void deleteRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testDeleteRecipe_8()
		throws Exception {

		Main.deleteRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.deleteRecipe(Main.java:92)
	}

	/**
	 * Run the void editRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testEditRecipe_1()
		throws Exception {

		Main.editRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.editRecipe(Main.java:111)
	}

	/**
	 * Run the void editRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testEditRecipe_2()
		throws Exception {

		Main.editRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.editRecipe(Main.java:111)
	}

	/**
	 * Run the void editRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testEditRecipe_3()
		throws Exception {

		Main.editRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.editRecipe(Main.java:111)
	}

	/**
	 * Run the void editRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testEditRecipe_4()
		throws Exception {

		Main.editRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.editRecipe(Main.java:111)
	}

	/**
	 * Run the void editRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testEditRecipe_5()
		throws Exception {

		Main.editRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.editRecipe(Main.java:111)
	}

	/**
	 * Run the void editRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testEditRecipe_6()
		throws Exception {

		Main.editRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.editRecipe(Main.java:111)
	}

	/**
	 * Run the void editRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testEditRecipe_7()
		throws Exception {

		Main.editRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.editRecipe(Main.java:111)
	}

	/**
	 * Run the void editRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testEditRecipe_8()
		throws Exception {

		Main.editRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.editRecipe(Main.java:111)
	}

	/**
	 * Run the void editRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testEditRecipe_9()
		throws Exception {

		Main.editRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.editRecipe(Main.java:111)
	}

	/**
	 * Run the void editRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testEditRecipe_10()
		throws Exception {

		Main.editRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.editRecipe(Main.java:111)
	}

	/**
	 * Run the void editRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testEditRecipe_11()
		throws Exception {

		Main.editRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.editRecipe(Main.java:111)
	}

	/**
	 * Run the void editRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testEditRecipe_12()
		throws Exception {

		Main.editRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.editRecipe(Main.java:111)
	}

	/**
	 * Run the void editRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testEditRecipe_13()
		throws Exception {

		Main.editRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.editRecipe(Main.java:111)
	}

	/**
	 * Run the void editRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testEditRecipe_14()
		throws Exception {

		Main.editRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.editRecipe(Main.java:111)
	}

	/**
	 * Run the void editRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testEditRecipe_15()
		throws Exception {

		Main.editRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.editRecipe(Main.java:111)
	}

	/**
	 * Run the void editRecipe() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testEditRecipe_16()
		throws Exception {

		Main.editRecipe();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.editRecipe(Main.java:111)
	}

	/**
	 * Run the String inputOutput(String) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testInputOutput_1()
		throws Exception {
		String message = "";

		String result = Main.inputOutput(message);

		// add additional test code here
		assertEquals(null, result);
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMain_1()
		throws Exception {
		String[] args = new String[] {};

		Main.main(args);

		// add additional test code here
	}

	/**
	 * Run the void mainMenu() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMainMenu_1()
		throws Exception {

		Main.mainMenu();

		// add additional test code here
	}

	/**
	 * Run the void mainMenu() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMainMenu_2()
		throws Exception {

		Main.mainMenu();

		// add additional test code here
	}

	/**
	 * Run the void mainMenu() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMainMenu_3()
		throws Exception {

		Main.mainMenu();

		// add additional test code here
	}

	/**
	 * Run the void mainMenu() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMainMenu_4()
		throws Exception {

		Main.mainMenu();

		// add additional test code here
	}

	/**
	 * Run the void mainMenu() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMainMenu_5()
		throws Exception {

		Main.mainMenu();

		// add additional test code here
	}

	/**
	 * Run the void mainMenu() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMainMenu_6()
		throws Exception {

		Main.mainMenu();

		// add additional test code here
	}

	/**
	 * Run the void mainMenu() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMainMenu_7()
		throws Exception {

		Main.mainMenu();

		// add additional test code here
	}

	/**
	 * Run the void mainMenu() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMainMenu_8()
		throws Exception {

		Main.mainMenu();

		// add additional test code here
	}

	/**
	 * Run the void mainMenu() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMainMenu_9()
		throws Exception {

		Main.mainMenu();

		// add additional test code here
	}

	/**
	 * Run the void mainMenu() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMainMenu_10()
		throws Exception {

		Main.mainMenu();

		// add additional test code here
	}

	/**
	 * Run the void mainMenu() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMainMenu_11()
		throws Exception {

		Main.mainMenu();

		// add additional test code here
	}

	/**
	 * Run the void mainMenu() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMainMenu_12()
		throws Exception {

		Main.mainMenu();

		// add additional test code here
	}

	/**
	 * Run the void mainMenu() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMainMenu_13()
		throws Exception {

		Main.mainMenu();

		// add additional test code here
	}

	/**
	 * Run the void mainMenu() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMainMenu_14()
		throws Exception {

		Main.mainMenu();

		// add additional test code here
	}

	/**
	 * Run the void mainMenu() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMainMenu_15()
		throws Exception {

		Main.mainMenu();

		// add additional test code here
	}

	/**
	 * Run the void mainMenu() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMainMenu_16()
		throws Exception {

		Main.mainMenu();

		// add additional test code here
	}

	/**
	 * Run the void makeCoffee() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMakeCoffee_1()
		throws Exception {

		Main.makeCoffee();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.makeCoffee(Main.java:217)
	}

	/**
	 * Run the void makeCoffee() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMakeCoffee_2()
		throws Exception {

		Main.makeCoffee();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.makeCoffee(Main.java:217)
	}

	/**
	 * Run the void makeCoffee() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMakeCoffee_3()
		throws Exception {

		Main.makeCoffee();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.makeCoffee(Main.java:217)
	}

	/**
	 * Run the void makeCoffee() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMakeCoffee_4()
		throws Exception {

		Main.makeCoffee();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.makeCoffee(Main.java:217)
	}

	/**
	 * Run the void makeCoffee() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMakeCoffee_5()
		throws Exception {

		Main.makeCoffee();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.makeCoffee(Main.java:217)
	}

	/**
	 * Run the void makeCoffee() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMakeCoffee_6()
		throws Exception {

		Main.makeCoffee();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.makeCoffee(Main.java:217)
	}

	/**
	 * Run the void makeCoffee() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMakeCoffee_7()
		throws Exception {

		Main.makeCoffee();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.makeCoffee(Main.java:217)
	}

	/**
	 * Run the void makeCoffee() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Test
	public void testMakeCoffee_8()
		throws Exception {

		Main.makeCoffee();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.NullPointerException
		//       at edu.towson.cis.cosc442.project4.coffeemaker.Main.makeCoffee(Main.java:217)
	}

	/**
	 * Perform pre-test initialization.
	 *
	 * @throws Exception
	 *         if the initialization fails for some reason
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@Before
	public void setUp()
		throws Exception {
		// add additional set up code here
	}

	/**
	 * Perform post-test clean-up.
	 *
	 * @throws Exception
	 *         if the clean-up fails for some reason
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	@After
	public void tearDown()
		throws Exception {
		// Add additional tear down code here
	}

	/**
	 * Launch the test.
	 *
	 * @param args the command line arguments
	 *
	 * @generatedBy CodePro at 3/22/18 1:54 PM
	 */
	public static void main(String[] args) {
		new org.junit.runner.JUnitCore().run(MainTest.class);
	}
}